contracts/Exchange.sol

```diff
@@ -276,8 +260,13 @@ contract Exchange is
             ) = (timestamp, fundingGrowthGlobal.twPremiumX96, fundingGrowthGlobal.twPremiumDivBySqrtPriceX96);
 
             emit FundingUpdated(baseToken, markTwap, indexTwap);
+        }
 
+        // last tick will be stopped updating once the market is being paused
+        uint256 lastTickUpdatedTimestamp = _lastTickUpdatedTimestampMap[baseToken];
+        if (lastTickUpdatedTimestamp == 0 || timestamp >= lastTickUpdatedTimestamp.add(_TICK_SNAPSHOT_INTERVAL)) {
```

Comment:
In practice, `timestamp >= lastTickUpdatedTimestamp.add(_TICK_SNAPSHOT_INTERVAL)` will always be true if  `lastTickUpdatedTimestamp == 0` so I think we can remove the first condition!

Reply:
Fixed in [2b75026](https://github.com/perpetual-protocol/perp-curie-contract/pull/63/commits/2b75026d05b8f70d2c57b1abbd6935fcb393c65d)